config/default.json

```diff
@@ -8,6 +8,7 @@
   "merkleDepthLimit": 0,
   "minStreamCount": 1,
   "anchorLauncherUrl": "http://localhost:8001",
+  "maxAnchoringDelayMS": 43200000,
```

Comment:
@oed how would you feel about making the max anchoring window be 6 hours instead of 12.  It will still save us a lot of work/money compared to what we're doing now, but give us a bit more wiggle room to make sure that the requests get anchored within the 24 hour CACAO window.

Reply:
also, do we do anything to prevent applications from setting the CACAO expiration to *less than* 24 hours?  If not that could break a lot of the guarantees we're aiming for